src/domain/dao.ts

```diff
@@ -42,23 +42,11 @@ export function insertNewDAO(
   dao.nativeToken = nativeTokenAddress.toHex();
   dao.nativeReputation = nativeReputationAddress.toHex();
   dao.reputationHoldersCount = BigInt.fromI32(0);
-  dao.register = 'na';
   saveDAO(dao);
 
   return dao;
 }
 
-export function register(
```

Comment:
why is this removed?

Reply:
Beacuse we don't have DAORegistry with the new Arc.

Reply:
DAORegistry was never part of arc . it comes from arc-hive repo https://github.com/daostack/arc-hive/blob/master/contracts/DAORegistry.sol .
we need that.

Reply:
Oh ok. I thought it will be removed with the new Architecture.